File: insertSort.py
def insertSort(array):

    for i in range(1, len(array)):
        
        n = i
        while (n > 0) and (array[n] < array[n-1]):
            temp = array[n-1]
            array[n-1] = array[n]
            array[n] = temp

            n -= 1

def main():

    inputs = [[5, 4, 2, 1, 3]]

    for input in inputs:
        print(input)
        insertSort(input)
        print(input)

main()

File: InsertSort.java
public class InsertSort{

    int[] insertSort(int[] array){

        for (int i = 1; i < array.length; i++){

            int n = i;
            while ((n > 0) && (array[n - 1] > array[n])){
                
                int temp = array[n - 1];
                array[n - 1] = array[n];
                array[n] = temp;

                n -= 1;
            }
        }

        return array;
    }
    

    public static void main(String[] args){
        InsertSort example = new InsertSort();
        int[] x = {6, 5, 1, 2, 3, 8, 7};
        System.out.println(example.insertSort(x)[0]);
        System.out.println(example.insertSort(x)[1]);
        System.out.println(example.insertSort(x)[2]);
        System.out.println(example.insertSort(x)[5]);
        System.out.println(example.insertSort(x)[6]);
    }
}